pkg/v1/providers/ytt/09_miscellaneous/08_store_data_values.yaml

```diff
@@ -44,6 +44,7 @@ metadata:
   labels:
     tkg.tanzu.vmware.com/cluster-name: #@ data.values.CLUSTER_NAME
     clusterctl.cluster.x-k8s.io/move: ""
+    tkg.tanzu.vmware.com/testnoncc: test
```

Comment:
what is this used for?

Reply:
please see the PR description. This is only included for demo purpose to show the resultant diffs, and will be removed before merging.